### ISSUE TITLE
Hello Khulna!

### ISSUE THREAD

**Issue by masnun** [nodeschool/khulna#1]
Just launched the website -- http://nodeschool.io/khulna/ :) Now let's plan for some meetups and online sessions.

**Comment by mehedijaman** [comment]
I am in.

**Comment by sajibsrs** [comment]
I'm in too :)

**Comment by suhel-fr** [comment]
I'm from Sylhet should I join?

**Comment by masnun** [comment]
@getsuhel You might be able to join our online sessions.

**Comment by asadlive84** [comment]
Thanks for your great idea.I am living Morrelgonj, Bagerhat.

**Comment by manashmandal** [comment]
I am currently studying at KUET. I am really interested in learning NodeJs. I have basic knowledge on javascript. Great initiative. Maybe we can meet sometimes. :+1:

**Comment by hassanurrakib** [comment]
Want to learn the node badly! :(

**Comment by Sahariar** [comment]
I am in Also :)

**Comment by suhel-fr** [comment]
Thank you @masnun  bhaia

**Comment by masnun** [comment]
Hey guys, awesome! How about a meetup? Let's meet and do a workshop together?

**Comment by manashmandal** [comment]
Saturday or Friday would be awesome. Cause, this week is the last one of my current semester, have a tight schedule next week.

**Comment by masnun** [comment]
Saturday afternoon is fine with me.

On Wednesday, March 11, 2015, Manash Mandal notifications@github.com
wrote:

> Saturday or Friday would be awesome. Cause, this week is the last one of
> my current semester, have a tight schedule next week.
> 
> ## 
> 
> Reply to this email directly or view it on GitHub
> https://github.com/nodeschool/khulna/issues/1#issuecomment-78200794.

## 

Abu Ashraf Masnun | +8801711960803 | http://masnun.me

**Comment by Sahariar** [comment]
Saturday Afternoon also Fine with me.

**Comment by Sahariar** [comment]
any update about meetup or workshop?

**Comment by masnun** [comment]
Hi Shahriar vai,

We need to meet and discuss a possible meetup date :)

On Wed, Mar 18, 2015 at 6:11 PM, Sahariar Kabir notifications@github.com
wrote:

> any update about meetup or workshop?
> 
> ## 
> 
> Reply to this email directly or view it on GitHub
> https://github.com/nodeschool/khulna/issues/1#issuecomment-82940976.

## 

Abu Ashraf Masnun | +8801711960803 | http://masnun.me

**Comment by Sahariar** [comment]
HI Masnun vai,

Where should we meet and discuss ?? 
:)

**Comment by masnun** [comment]
We can meet in any restaurants you prefer :)

On Fri, Mar 20, 2015 at 12:27 PM, Sahariar Kabir notifications@github.com
wrote:

> HI Masnun vai,
> 
> Where should we meet and discuss ??
> :)
> 
> ## 
> 
> Reply to this email directly or view it on GitHub
> https://github.com/nodeschool/khulna/issues/1#issuecomment-83929419.

## 

Abu Ashraf Masnun | +8801711960803 | http://masnun.me

**Comment by Sahariar** [comment]
Yes I think we can meet on Shwarma House,Hotel Tiger Garden International,Shibbari, Khulna.

what you think about it Please let me Know??

**Comment by masnun** [comment]
Sure, I go there regularly.

On Wed, Mar 25, 2015 at 10:45 PM, Sahariar Kabir notifications@github.com
wrote:

> Yes I think we can meet on Shwarma House,Hotel Tiger Garden
> International,Shibbari, Khulna.
> 
> what you think about it Please let me Know??
> 
> ## 
> 
> Reply to this email directly or view it on GitHub
> https://github.com/nodeschool/khulna/issues/1#issuecomment-86114789.

## 

Abu Ashraf Masnun | +8801711960803 | http://masnun.me

**Comment by Sahariar** [comment]
oh that's Great. Which day and what time we should meet?

:+1:

**Comment by masnun** [comment]
You can choose. Saturday and Sunday afternoon works best for me.

On Thu, Mar 26, 2015 at 1:08 AM, Sahariar Kabir notifications@github.com
wrote:

> oh that's Great. Which day and what time we should meet?
> 
> [image: :+1:]
> 
> ## 
> 
> Reply to this email directly or view it on GitHub
> https://github.com/nodeschool/khulna/issues/1#issuecomment-86176299.

## 

Abu Ashraf Masnun | +8801711960803 | http://masnun.me

**Comment by Sahariar** [comment]
I think this Sunday Afternoon will be great :)

**Comment by masnun** [comment]
OK, call me before coming. 01711 960803 :)

On Thu, Mar 26, 2015 at 6:33 PM, Sahariar Kabir notifications@github.com
wrote:

> I think this Sunday Afternoon will be great :)
> 
> ## 
> 
> Reply to this email directly or view it on GitHub
> https://github.com/nodeschool/khulna/issues/1#issuecomment-86490723.

## 

Abu Ashraf Masnun | +8801711960803 | http://masnun.me

**Comment by 2farhann** [comment]
@masnun @Sahariar Can anybody else join the Sunday meeting?
I'm interested.

**Comment by masnun** [comment]
Sure, but we met yesterday. We're trying to gather enough interested
participants before we arrange an event.

On Sat, Mar 28, 2015 at 3:21 PM, 2farhann notifications@github.com wrote:

> @masnun https://github.com/masnun @Sahariar
> https://github.com/Sahariar Can anybody else join the Sunday meeting?
> I'm interested.
> 
> ## 
> 
> Reply to this email directly or view it on GitHub
> https://github.com/nodeschool/khulna/issues/1#issuecomment-87197246.

## 

Abu Ashraf Masnun | +8801711960803 | http://masnun.me

**Comment by 2farhann** [comment]
@masnun How can I start learning js here?

**Comment by masnun** [comment]
http://nodeschool.io - there are fantastic tools on this page.

On Sat, Mar 28, 2015 at 3:34 PM, 2farhann notifications@github.com wrote:

> @masnun https://github.com/masnun How can I start learning js here?
> 
> ## 
> 
> Reply to this email directly or view it on GitHub
> https://github.com/nodeschool/khulna/issues/1#issuecomment-87197851.

## 

Abu Ashraf Masnun | +8801711960803 | http://masnun.me

**Comment by 2farhann** [comment]
Thank you so much.

**Comment by 2farhann** [comment]
![javascripting](https://cloud.githubusercontent.com/assets/11692413/6880960/70d13072-d573-11e4-87d4-07d7f9328ab7.PNG)
Just completed the javascripting. It was really fun. Thanks @masnun for referring the site. I'll start the Functional JavaScript soon.

**Comment by Sahariar** [comment]
HI @2farhann,
 it's Great to know that you Completed your 1st way to Javascript lesson. Please share What you like most about JS ?
 :)

**Comment by masnun** [comment]
Awesome man! Congratulations! :)

On Sun, Mar 29, 2015 at 4:08 AM, Sahariar Kabir notifications@github.com
wrote:

> HI @2farhann https://github.com/2farhann,
> it's Great to know that you Completed your 1st way to Javascript. Please
> share What you like most about JS ?
> :)
> 
> ## 
> 
> Reply to this email directly or view it on GitHub
> https://github.com/nodeschool/khulna/issues/1#issuecomment-87321834.

## 

Abu Ashraf Masnun | +8801711960803 | http://masnun.me

**Comment by 2farhann** [comment]
Thanks.

As a front-end web developer I had to work with jQuery. jQuery is a library of JS but I don't know much about JS. And I am very curious to know how JS manipulate DOM what it is capable of.

I like Ajax most.

@masnun I am stuck in third problem of Functional JavaScript. It's about Array#map. Can you please help me!!!

**Comment by masnun** [comment]
Sure, send me the problem.

On Sun, Mar 29, 2015 at 4:20 PM, 2farhann notifications@github.com wrote:

> Thanks.
> 
> As a front-end web developer I had to work with jQuery. jQuery is a
> library of JS but I don't know much about JS. And I am very curious to know
> how JS manipulate DOM what it is capable of.
> 
> I like Ajax most.
> 
> @masnun https://github.com/masnun I am stuck in third problem of
> Functional JavaScript. It's about Array#map. Can you please help me!!!
> 
> ## 
> 
> Reply to this email directly or view it on GitHub
> https://github.com/nodeschool/khulna/issues/1#issuecomment-87387850.

## 

Abu Ashraf Masnun | +8801711960803 | http://masnun.me

**Comment by 2farhann** [comment]
![pro](https://cloud.githubusercontent.com/assets/11692413/6885329/4673c27a-d63d-11e4-8b1e-3754fc7a9781.PNG)

Here it is.

**Comment by masnun** [comment]
Use Array.prototype.map() function with a callback that would double each
item.

See this --
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/map

PS: For off topic discussion, please create a new thread :)

On Sun, Mar 29, 2015 at 5:59 PM, 2farhann notifications@github.com wrote:

> [image: pro]
> https://cloud.githubusercontent.com/assets/11692413/6885329/4673c27a-d63d-11e4-8b1e-3754fc7a9781.PNG
> 
> Here it is.
> 
> ## 
> 
> Reply to this email directly or view it on GitHub
> https://github.com/nodeschool/khulna/issues/1#issuecomment-87403528.

## 

Abu Ashraf Masnun | +8801711960803 | http://masnun.me

**Comment by az7arul** [comment]
@nodeschool/khulna doing anything on international day ?

**Comment by rahmatelinks** [comment]
i want to learn node.js but may be i should know the js first . @masnun vai do you want to teach me js  in live. really i'm so much interested .

**Comment by masnun** [comment]
@rahmatelinks  Yes, we're yet to arrange our first workshop. We shall do one very soon.
